packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -74,12 +76,16 @@ class ConfigScreenComponent extends React.Component<any, any> {
 		this.setState({ settings: this.props.settings });
 	}
 
-	public componentDidMount() {
+	public async componentDidMount() {
 		if (this.props.defaultSection) {
 			this.setState({ selectedSectionName: this.props.defaultSection }, () => {
 				void this.switchSection(this.props.defaultSection);
 			});
 		}
+
+		const fonts = (await (window as any).queryLocalFonts()).map((font: any) => font.family);
```

Comment:
I assume there's a type for this somewhere?

If not please define the properties that you need to replace "font: any"

Reply:
It has a type in the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/FontData). But it's not defined in TS, which makes sense as the function in the first place is not defined :" So I defined it myself. 5113b57